osbenchmark/worker_coordinator/runner.py

```diff
@@ -1101,7 +1101,7 @@ async def __call__(self, opensearch, params):
                                      )
 
     def __repr__(self, *args, **kwargs):
-        return "put-pipeline"
+        return opensearch.ingest.put_pipeline.return_value = as_future()
```

Comment:
We should change this line back as it might be impacting the CI runs

Reply:
not sure why thats left, thought I changed all of them. will change today, sorry about that.

Reply:
No worries!